### PR TITLE
Tappx Bid Adapter: optional ext on request

### PR DESCRIPTION
**Pull request by prebidtappx**
## Type of change

- [x] Feature

## Description of change

This change is going to allow send extra data to tappx via prebid.js using "ext" property in the bidder.

### CODE REVIEW

**Comment by patmmccann** [comment]
Please pull prebid master into your fork and branch so as to not affect outbrain

**Comment by prebidtappx** [comment]
> Please pull prebid master into your fork and branch so as to not affect outbrain

Hi! We had the same problem in other branch, the pull doesn't work (we're not sure why), but if we change manually seems OK.
Thanks for all!

File: modules/tappxBidAdapter.js [hunk 1]
```diff
@@ -336,7 +338,9 @@ function buildOneRequest(validBidRequests, bidderRequest) {
   payload.params = params;
   payload.regs = regs;
   // < Payload
-
+  utils.logMessage('---------------------');
+  utils.logMessage(JSON.stringify(payload));
```
Comment (patmmccann) [inline modules/tappxBidAdapter.js:342]: why

Comment (patmmccann) [reply modules/tappxBidAdapter.js:342]: nm, see this is behind debug mode now, sorry

**Review by harpere — APPROVED** [review]

**Pull request merged**